File: Code/java/largest.java
import java.util.Scanner;

class Largest
{
    public static void main(String[] args)
    {
        int limit, i;
        float input, largest = -999999999;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Limit: ");
        limit = scanner.nextInt();
        for (i = 0; i < limit; i++)
        {
            System.out.println("Enter Input: ");
            input = scanner.nextFloat();
            if (i == 0 || largest < input)
                largest = input;
        }

        System.out.println(largest);

        scanner.close();
    }
}

File: Code/cpp/largest.cpp
#include<iostream>
using namespace std;
int main()
{
    int limit, i;
    float input, largest;

    cout << "Enter Limit: ";
    cin >> limit;
    for(i = 0; i < limit; i++)
    {
        cout << "Enter Input: ";
        cin >> input;
        if (i == 0 || largest < input)
            largest = input;
    }
    
    cout << largest;
    return 0;
}